content/embeds/tryout-redisearch-2.md

```diff
@@ -0,0 +1,93 @@
+## Creating indexes
+
+Let's create a new index called "myIdx". When you define the index, you
+must pass in the structure of the data you are adding to the index.
+In this example, we have four things we input: title, body, url, and
+value. In this example, we have three TEXT and one NUMERIC values. The
+title has a weight of 5.0.
+
+1. Connect to redis.
+
+    ```sh
+    $ redis-cli -p 12543
+    127.0.0.1:12543>
+    ```
+
+1. Create the schema with the command:
+
+    ```sh
+    127.0.0.1:12543> FT.CREATE myIdx PREFIX 1 "doc:" SCORE_FIELD "value" SCHEMA title TEXT body TEXT url TEXT value NUMERIC
+    ```
+
+    This command indexes the hashes that have the prefix "doc".
+    By default, all hashes get a score of 1.0 but here the index uses the `value` field as the score for this hash.
+
+{{< note >}}
+For databases in a Redis Cloud Essentials subscription, you need to add the index name to the document key as a tag to make sure that the index and the documents are located on the same shard:
+
+```sh
+FT.CREATE myIdx ...
+HSET myIdx {myIdx}:docid ...
+```
+
+{{< /note >}}
+
+### Add data to test index
+
+Now add some data to this index. Here we add an object with the key
```

Comment:
s/an object/a hash/

Here we're adding a hash and storing it at the key "doc:1".

Also, please use a realistic key name. "doc1" is not a realistic key name. We typically separate parts of key names with colons. So "doc:1" is more realistic.

"my favorite object" - this reads really oddly. Can create something more real-world sounding? We need our examples to feel concrete. This makes for more engaging docs.

Reply:
For example, we could pretend we're indexing an actual Redis Labs blog post...